nemoguardrails/library/cleanlab/flows.co

```diff
@@ -0,0 +1,17 @@
+"""
+https://cleanlab.ai/tlm/
+
+https://help.cleanlab.ai/tutorials/tlm/#how-does-the-tlm-trustworthiness-score-work
+
+"""
+
+define subflow cleanlab trustworthiness
+  """Guardrail based on the trustworthiness score."""
+  $result = execute call cleanlab api
+
+  if $result.trustworthiness_score < 0.9
```

Comment:
```suggestion
  if $result.trustworthiness_score < 0.6
```

Reply:
not sure what impact this has, but I recommend default of 0.6 for the alert threshold